### PR TITLE
Animate_1

### LINKED ISSUE #198: Animate_1
Link to NCL code: https://www.ncl.ucar.edu/Applications/Scripts/animate_1.ncl
Link to NCL Projection: https://www.ncl.ucar.edu/Applications/Images/animate_1_1_lg.png

### PR DESCRIPTION
**Pull request by michaelavs**
closes #198 
Original NCL Plot (1/30):
https://www.ncl.ucar.edu/Applications/Images/animate_1_1_lg.png

Generated Plot (1/30):
image

### CODE REVIEW

**Comment by michaelavs** [comment]
@erogluorhan I've been looking into how to display the gif in Sphinx this morning and haven't gotten too far. However, I have also been trying to at least display the first projection (day 1) like what the NCL example has, but it appears Sphinx doesn't process the images for display from the code because (I'm assuming) they are being stored locally, even though I call plt.show() in the code which would imply I should see either all 30 of them (as I do in my Spyder kernel) or I should at least see the last one generated. I've tried removing the plt.close() command, but then each projection becomes subsetted in the next and its a mess. Do you have any insight on why this may be happening for just the images specifically? On that same subject, I wanted to also point out that all 30 png files are saving to the "Animations" directory along with saving in my local "Downloads" folder. Is this something I should be worried about for this PR?

File: Plots/Animations/NCL_animate_1.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+"""
+NCL_animate_1.py
+===============
```
Comment (anissa111) [inline Plots/Animations/NCL_animate_1.py:3]:
```suggestion
================
```

File: Plots/Animations/NCL_animate_1.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+   - Creating animations
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/animate_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/animate_1_1_lg.png
+
+Please note:
+    This code will create a .gif file locally. In order to view the gif, you 
+    will need to right click on the .gif file in your local directory, then 
+    select 'open with' then select a web browser. This has been tested using
```
Comment (anissa111) [inline Plots/Animations/NCL_animate_1.py:14]: Depending on operating system and preview settings, this does not necessarily need to be opened in a web browser. I don't think we have to tell them how to open it beyond that the script will create a gif file locally.

File: Plots/Animations/NCL_animate_1.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    Google Chrome, Safari, and Firefox. 
+"""
+###############################################################################
+# Import packages:
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from matplotlib import pyplot as plt
+import numpy as np
+import xarray as xr
+from PIL import Image, ImageDraw
```
Comment (anissa111) [inline Plots/Animations/NCL_animate_1.py:24]:
```suggestion
from PIL import Image
```

**Review by anissa111 — CHANGES_REQUESTED** [review]
There are a few little comment here, but I think there may be a better way of accomplishing this. Since we're not trying to get this to display on RTD anymore, I think we can skip some of the middle steps of generating and saving each image and then re-grabbing them and putting them in a gif. 

Also, the call at the end to display the image does not seem to be doing anything for me on my local, I'm not sure it's needed since we aren't trying to display anything necessarily.

File: Plots/Animations/NCL_animate_1.py (hunk 2 above)
Comment (michaelavs) [reply Plots/Animations/NCL_animate_1.py]: Just updated, let me know what you think!

File: Plots/Animations/README.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+.. _animation_examples:
+
+.. _animation-examples-index:
+
+Animate
+===============
```
Comment (anissa111) [inline Plots/Animations/README.rst:6]:
```suggestion
Animate
=======
```

File: Plots/Animations/NCL_animate_1.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+###############################################################################
+# Create animation:
+
+fig = plt.figure(figsize=(10, 8))
+# Generate axes using Cartopy and draw coastlines
+ax = plt.axes(projection=ccrs.PlateCarree(central_longitude=150))
+ax.coastlines(linewidths=0.5)
+
+# Set extent to include latitudes from 34 to 52 and longitudes from 128
+# to 144
```
Comment (anissa111) [inline Plots/Animations/NCL_animate_1.py:47]:
```suggestion
```

Comment inaccurate and I think it can be removed

**Review by anissa111 — COMMENTED** [review]
This looks good, haha. For the other reviewers, I recommend looking at the [animate_1 RTD branch](https://geocat-examples.readthedocs.io/en/animate_1/gallery/Animations/NCL_animate_1.html#sphx-glr-gallery-animations-ncl-animate-1-py), if for no other reason than I'm so happy that a gif is finally able to be displayed. 

I've included a few super small last minute changes below, but after that, it should be good to go!

**Review by erogluorhan — APPROVED** [review]
This looks great, and I am so glad your efforts worked out in RTD side :)

**Review by anissa111 — APPROVED** [review]
I'm very happy to be able to approve this! The tests got stuck on that one wrf example, I think if they're re-run, it will be fine.

**Pull request merged**